### PR TITLE
scaling methods added

### PR DESCRIPTION
**Pull request by jobo322**

### CODE REVIEW

File: src/pre/scaling.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+
+const Matrix = require('ml-matrix');
```
Comment (maasencioh) [inline src/pre/scaling.js]: first you need to import this package

File: src/pre/scaling.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+function vastScaling(dataSet, mean, std) {
+    var means = mean || Stat.matrix.mean(dataSet);
+    var std = std || Stat.matrix.standardDeviation(dataSet, means, true);
+    var result = autoScaling(dataSet, means, std);
+    result.mulColumnVector(means)
+    return result.divColumnVector(std);
+}
+
+function centering(dataSet, mean) {
```
Comment (maasencioh) [inline src/pre/scaling.js]: centering it's not exactly a scaling method, so I'll split this one

**Review by maasencioh — CHANGES_REQUESTED** [review]
This is an old repository, so I'll work on an update of it, but it's important to check first that passes all the tests, that by the way are missing

**Comment by maasencioh** [comment]
ping @jobo322

File: src/pre/scaling.js (hunk 2 above)
Comment (jobo322) [reply src/pre/scaling.js]: where can I put it?

File: src/pre/centering.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function centering(dataSet, mean) {
```
Comment (lpatiny) [inline src/pre/centering.js]: center

Comment (lpatiny) [reply src/pre/centering.js]: and export it in index.js

File: src/index.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 'use strict';
 
 exports.normal = require('./pre/normal');
-exports.select = require('./pre/select');
-exports.discret = require('./pre/discret');
+exports.scaling = require('./pre/scaling');
```
Comment (lpatiny) [inline src/index.js]: scale

**Pull request merged**